handbook/brand.md

```diff
@@ -58,11 +58,11 @@ While we encourage and equip our writers to succeed by themselves in editing que
 
 The following is our handy guide to editor bliss at Fleet, but first, let's start by looking at some common content types that require an editor pass. 
 
-- Recently merged Pull Requests to the handbook and docs.
-- Articles, release posts, and press releases.
-- Social media posts
+- How do you find recently merged Pull Requests for the handbook and docs?
```

Comment:
These suggestions change the meaning of the sentence above. They are supposed to list the types of content that require an editor pass, not an index of the sections below.